packages/vscode-extension-pmml-editor/webpack.config.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require("path");
+const pfWebpackOptions = require("@kogito-tooling/patternfly-base/patternflyWebpackOptions");
+const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
+
+const commonConfig = {
+    mode: "development",
```

Comment:
Since I am seeing 4 spaces per indentation-level (other files have 2), 
I believe this file is not being formatted with `prettier`.

Reply:
Oops.. fixed. `prettier` now configured in my IDE.